<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="menuList">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <item>
        <property name="text">
         <string>Arduino</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>User interface</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Code editor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Compiler</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_2">
          <widget class="QLabel" name="lblName_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>451</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Arduino options:</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="uploadTimeout">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>100</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>600</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>100</y>
             <width>231</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Upload timeout (seconds)</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="userLibraries">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>160</y>
             <width>391</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>140</y>
             <width>131</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>User libraries path:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>190</y>
             <width>411</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>(If you have folders with your own libraries, put their paths here, separated by a semi-colon)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="arduinoInstall">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>391</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>231</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Arduino install:</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnLoadCore">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>60</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnLoadUserLibs">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>160</y>
             <width>21</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
          <zorder>label_4</zorder>
          <zorder>lblName_2</zorder>
          <zorder>uploadTimeout</zorder>
          <zorder>userLibraries</zorder>
          <zorder>label_5</zorder>
          <zorder>label_6</zorder>
          <zorder>arduinoInstall</zorder>
          <zorder>label_8</zorder>
          <zorder>btnLoadCore</zorder>
          <zorder>btnLoadUserLibs</zorder>
         </widget>
         <widget class="QWidget" name="page_3">
          <widget class="QLabel" name="lblName_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>451</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>User interface</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="embedMenu">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>301</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Embed main menu in the toolbar</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_4">
          <widget class="QLabel" name="colorPicked">
           <property name="geometry">
            <rect>
             <x>380</x>
             <y>220</y>
             <width>61</width>
             <height>21</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QComboBox" name="cbThemes">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>321</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Dark</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>90</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Font size</string>
           </property>
          </widget>
          <widget class="QFontComboBox" name="editorFontName">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>110</y>
             <width>321</width>
             <height>22</height>
            </rect>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="fontFilters">
            <set>QFontComboBox::AllFonts</set>
           </property>
          </widget>
          <widget class="QLabel" name="lblName_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>451</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Color scheme</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btnColorPicker">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>220</y>
             <width>91</width>
             <height>21</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>Choose the background color of the editor</string>
           </property>
           <property name="text">
            <string>Editor color</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="editorFontSize">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>110</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>96</number>
           </property>
          </widget>
          <widget class="QLabel" name="lblName_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>241</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Editor font</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblName_7">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>451</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Code editor</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="cbBraceHighlight">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>301</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Highlight matching/unmatching braces (*)</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblName_8">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>230</y>
             <width>451</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>(*) This may cause some flickering on Windows. Will be fixed on future versions</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page">
          <widget class="QLabel" name="lblName_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>451</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Compiler options</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="hideWarnings">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>321</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Hide compiler warning messages</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnOk">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnApply">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
